extensions/data-workspace/package.nls.json

```diff
@@ -11,5 +11,6 @@
 	"projects-view-no-project-content": "No projects found in current workspace.\n[Create new](command:projects.new)\n[Open existing](command:projects.openExisting)\nTo learn more about SQL Database projects [read our docs](https://aka.ms/azuredatastudio-sqlprojects).\n",
 	"open-existing-command": "Open existing",
 	"projects.defaultProjectSaveLocation": "Full path to folder where new projects are saved by default.",
-	"projects.showNotAddedProjectsInWorkspacePrompt": "Always show information message when the current workspace folders contain projects that have not been added to the workspace's projects."
+	"projects.showNotAddedProjectsInWorkspacePrompt": "Always show information message when the current workspace folders contain projects that have not been added to the workspace's projects.",
+	"manage-project-command": "Manage"
```

Comment:
is the verb "Manage" from the mockups?  Now that I'm seeing it in person, I wonder if it has too similar of a meaning to "Properties", which will likely be getting added later.  Something more boring like "Open Dashboard" might be less confusing down the line.

Reply:
"Manage" basically comes from the existing dashboards (server/db and BDC dashboard). Hence, didn't think on the lines of similarities with "Properties". 
I will keep it as-is for now, but surely discuss this with Raden separately.